### ISSUE TITLE
@conda-forge-admin, please update version

### ISSUE THREAD

**Issue by t-b** [conda-forge/gcovr-feedstock#16]

**Comment by t-b** [comment]
Fixed dependencies for 8.2:

```diff
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index fc718a0..22d6196 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -23,14 +23,14 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.9
     - pip
-    - setuptools
   run:
-    - python >=3.7
-    - pygments
+    - python >=3.9
+    - pygments >=2.13.0
     - jinja2
     - lxml
+    - tomli >=1.1.0=(py39_0|py310_0)
 
 test:
   imports:
```

**Comment by t-b** [comment]
Done in https://github.com/conda-forge/gcovr-feedstock/pull/17.
